tests/integration/targets/ndo_port_channel_interface/tasks/main.yml

```diff
@@ -0,0 +1,437 @@
+
+    - name: Assert port channel interface creation tasks
+      assert:
+        that:
+          - cm_create_new_port_channel_interface is changed
+          - cm_create_new_port_channel_interface.previous == {}
+          - cm_create_new_port_channel_interface.current.name == "ansible_port_channel_interface"
+          - cm_create_new_port_channel_interface.current.description == "Ansible Port Channel test"
+          - cm_create_new_port_channel_interface.current.node == "101"
+          - cm_create_new_port_channel_interface.current.memberInterfaces == "1/1"
+          - cm_create_new_port_channel_interface.current.interfaceDescriptions | length == 1
+          - cm_create_new_port_channel_interface.current.interfaceDescriptions.0.nodeID == "101"
+          - cm_create_new_port_channel_interface.current.interfaceDescriptions.0.interfaceID == "1/1"
+          - cm_create_new_port_channel_interface.current.interfaceDescriptions.0.description == "first Ansible interface test"
+          - nm_create_new_port_channel_interface is changed
+          - nm_create_new_port_channel_interface.previous == {}
+          - nm_create_new_port_channel_interface.current.name == "ansible_port_channel_interface"
+          - nm_create_new_port_channel_interface.current.description == "Ansible Port Channel test"
+          - nm_create_new_port_channel_interface.current.node == "101"
+          - nm_create_new_port_channel_interface.current.memberInterfaces == "1/1"
+          - nm_create_new_port_channel_interface.current.interfaceDescriptions | length == 1
+          - nm_create_new_port_channel_interface.current.interfaceDescriptions.0.nodeID == "101"
+          - nm_create_new_port_channel_interface.current.interfaceDescriptions.0.interfaceID == "1/1"
+          - nm_create_new_port_channel_interface.current.interfaceDescriptions.0.description == "first Ansible interface test"
+          - nm_create_new_port_channel_interface_again is not changed
+          - nm_create_new_port_channel_interface_again.previous == nm_create_new_port_channel_interface_again.current
+          - nm_create_new_port_channel_interface_again.current.name == "ansible_port_channel_interface"
+          - nm_create_new_port_channel_interface_again.current.description == "Ansible Port Channel test"
+          - nm_create_new_port_channel_interface_again.current.node == "101"
+          - nm_create_new_port_channel_interface_again.current.memberInterfaces == "1/1"
+          - nm_create_new_port_channel_interface_again.current.interfaceDescriptions | length == 1
+          - nm_create_new_port_channel_interface_again.current.interfaceDescriptions.0.nodeID == "101"
+          - nm_create_new_port_channel_interface_again.current.interfaceDescriptions.0.interfaceID == "1/1"
+          - nm_create_new_port_channel_interface_again.current.interfaceDescriptions.0.description == "first Ansible interface test"
+
+    # CREATION ERRORS
```

Comment:
Do we need to keep the pattern consistent and move error just before delete tasks, or it this okay?